build_runner/lib/src/generate/build_impl.dart

```diff
@@ -478,6 +478,8 @@ class BuildImpl {
     for (var input in allInputs) {
       if (_isValidInput(input)) {
         _inputsByPackage[input.package].add(input);
+      } else {
```

Comment:
I don't think we want this else, we just want to skip the invalid inputs at this stage. Users aren't trying to read inputs yet, we are just collecting the list of possible inputs (and skipping the invalid ones).

Also, given the globs above it doesn't seem like this should ever be hit? Maybe this whole section can be removed.